kernel/common/enumerable.rb

```diff
@@ -567,86 +567,110 @@ def first(n=undefined)
     nil
   end
 
-  def min
-    min = undefined
-    each do
-      o = Rubinius.single_block_arg
```

Comment:
Oh I wasn't condemning you for it, but when changing this anyway it would be nice to take care of cases like this :)